Kronos-Shifts-Connector/README.md

```diff
@@ -22,6 +22,7 @@ The points noted below are to be considered as best practices to properly levera
 * Teams and Scheduling groups must be created in Shifts App prior Teams to Department mapping step in Configuration Web App  
 * Done button on Configuration Web App should be used only for first time sync  
 * First time sync is expected to take longer time since it may sync data for larger time interval. The time would vary based on amount of data i.e. number of users, number of teams, number of entities (such as Shifts, Time Offs, Open Shifts etc.) to be synced and date span of the Time interval for which the sync is happening. So, it may take time to reflect this complete data in Shifts. Done button click will initiate background process to complete the sync  
+* FLWs are to be aware of which PayCodes they have access to for creating the Time Off requests. The paycodes are to be configured ahead of time in the Kronos WFC instances.
```

Comment:
Please call out supported paycodes in the current deployment as is.

Reply:
@aamini-ms - You can look at the latest commit as I have made the changes you have recommended